Project2015To2017.Console/Program.cs

```diff
@@ -51,6 +53,9 @@ private static void ConvertProject(Options options)
 				writer.Write(project, doBackup, progress);
 			}
 
+			System.Console.Out.Flush();
+
+			conversionOptions.AnalyzeOnly = true;
```

Comment:
I don't get this one, why do we need to set that here?

Reply:
Oh, that was part of next PR (analyzer). After writing we do full _read_ _&_ _analysis_ procedure again to account for changed locations of XML elements.